### DISCUSSION TITLE
OP Node sync stuck

### DISCUSSION THREAD

**Discussion by joslee7410** [ethereum-optimism/developers#138] [Node Operators]
**Describe the bug**
My OP node after upgrade to the latest version
- opnode: v1.4.0
- opgeth: v1.101304.2

It keep stuck at block "113634635" and show

`msg="no peers ready to handle block requests for more P2P requests for L2 block history" target=113,634,635 end=0x55b52d95841f6ecaaa31a35001f01526c06d9a35d771d57c10d22cb9a78b2898:113635626 current=113,635,410`

**System Specs:**
 - OS: ubuntu 20.04.4 LTS
 - 16 CPU(s), 30Gi Ram

**Comment by sbvegan** [comment]
@joslee7410 can you please provide your node configurations (i.e. the flags you pass into `op-node` and `op-geth`)? Is this a OP Mainnet node or OP Sepolia node?

**Reply by sbvegan** [reply]
I'm closing this for now, but if you can provide additional information, we can reopen and debug.
